DataFormats/Parameters/include/DataFormatsParameters/GRPObject.h

```diff
@@ -158,14 +159,17 @@ class GRPObject
   bool mUniformField = false;                 ///< uniformity of magnetic field
   float mBeamEnergyPerZ = 0.f;                ///< beam energy per charge (i.e. sqrt(s)/2 for pp)
 
+  int8_t mNominalL3Field = 0;        ///< Nominal L3 field deduced from mL3Current
+  bool mNominalL3FieldValid = false; ///< Has the field been computed (for caching)
```

Comment:
fine for me, but please make it 
```
int8_t mNominalL3Field = 0;        //!  Nominal L3 field deduced from mL3Current
bool mNominalL3FieldValid = false; //! Has the field been computed (for caching)
```
to not store on-line values.
Note that this object will be eventually discarded, @chiarazampolli splits is to a few objects according to data provenance: https://github.com/AliceO2Group/AliceO2/pull/7287

Reply:
Done. @chiarazampolli can you port also this addition, then?